src/com/pentalog/pentastagiu/homework2/shop/customermembership/MembershipType.java

```diff
@@ -0,0 +1,5 @@
+package com.pentalog.pentastagiu.homework2.shop.customermembership;
+
+public enum MembershipType {
```

Comment:
pt ca membership-ul e strans legat de Customer, ar fi putut fi adaugat in clasa Customer si sa il fi accesat asa Customer,MembershipType.GOLD